src/openrct2-ui/windows/RideList.cpp

```diff
@@ -901,15 +913,24 @@ namespace OpenRCT2::Ui::Windows
         void SortListByPredicate(const TSortPred& pred)
         {
             bool desc = _windowListSortDescending;
-            std::stable_sort(_rideList.begin(), _rideList.end(), [desc, pred](const auto& lhs, const auto& rhs) {
```

Comment:
stable sort no longer needed, previously it was handy because when changing the sorting key (e.g. going from status (default sort, alphebatic) to e.g. profit, when it has a tie in profit, it keep the relativ order it had before (alphabetic)

this made it not actually alphabetic because when you then revert order, or go to another non-alphabetic sort (e.g.from profit to intensity) the sort between ties is messy. now its just a normal sort, which is fine because secondary sort based on name is down below

Reply:
Honestly, I don't remember why I changed them to `std::stable_sort` in #24893. If it works fine without, I'm fine with that.